jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/MemoryMXBeanImpl.java

```diff
@@ -777,33 +777,53 @@ public String getGCMode() {
 	private native String getGCModeImpl();
 
 	/**
-	 * Returns the amount of CPU time spent in the GC by the master thread, in milliseconds.
+	 * Returns the amount of CPU time spent in the GC by the main thread, in milliseconds.
 	 *
 	 * @return CPU time used in milliseconds
-	 * @see #getGCMasterThreadCpuUsed()
+	 * @see #getGCMainThreadCpuUsed()
 	 */
-	private native long getGCMasterThreadCpuUsedImpl();
+	private native long getGCMainThreadCpuUsedImpl();
 
+/*[IF !Java16]*/
 	/**
 	 * {@inheritDoc}
 	 */
+	@Deprecated(forRemoval=true, since="15")
```

Comment:
I don't think this works for Java 8 (Java 8 won't compile), you need to use the processor.

Reply:
fixed.